File: Programming Fundamentals with Python/Functions/factorial_division.py
import math

def factorial_division(num1, num2):
    factorial_num1 = math.factorial(num1)
    factorial_num2 = math.factorial(num2)
    return factorial_num1 / factorial_num2


num1 = int(input())
num2 = int(input())

result = factorial_division(num1, num2)
print(f"{result:.2f}")

File: Programming Fundamentals with Java/Methods/FactorialDivision.java
import java.util.Scanner;

public class FactorialDivision 
{
    public static long factorial(int num)
    {
        long result = 1;
        for (int i = 1; i <= num; i++)
        {
            result *= i;
        }
        return result;
    }

    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        int num1 = Integer.parseInt(input.nextLine());
        int num2 = Integer.parseInt(input.nextLine());

        long factorialNum1 = factorial(num1);
        long factorialNum2 = factorial(num2);

        double result = factorialNum1 / factorialNum2;
        System.out.printf("%.2f", result);
    }
}
